modules/installation-custom-gcp-vpc.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+
+:_content-type: CONCEPT
+[id="installation-custom-gcp-vpc_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into existing subnets in an existing Virtual Private Cloud (VPC) in Google Cloud Platform (GCP). By deploying {product-title} into an existing GCP VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. If you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself, use this installation option.
+
+Because the installation program cannot know what other components are also in your existing subnets, it cannot choose subnet CIDRs and so forth on your behalf. You must configure networking for the subnets that you install your cluster to yourself.
+
+[id="installation-custom-gcp-vpc-requirements_{context}"]
+== Requirements for using your VPC
+
+The VPC’s CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines. The subnet CIDR blocks must belong to the machine CIDR that you specify.
+
+The installation program no longer creates the following components:
+
+* NAT gateways
+* Subnets
+* Route tables
+* VPC network
+
+[id="installation-custom-gcp-vpc-validation_{context}"]
+== VPC validation
+
+To ensure that the subnets that you provide are suitable, the installation program confirms the following data:
+
+* All the subnets that you specify exist.
+* You provide private subnets.
+* The subnet CIDRs belong to the machine CIDR that you specified.
+
+[id="installation-about-custom-gcp-permissions_{context}"]
+== Division of permissions
+
+
+[id="installation-custom-gcp-vpc-isolation_{context}"]
+== Isolation between clusters
```

Comment:
Also empty, please confirm.

Reply:
I added the content can you please check if it holds true for GCP?